doc/source/development_and_release_processes.rst

```diff
@@ -0,0 +1,203 @@
+      not merged.
+
+The stable *auto-generated packages* and release candidate *idiomatic packages* can now be used for the final release of
+internal packages. It is recommended to make any closed-source *idiomatic package* release candidate dependencies
+flexible enough such that they match subsequent stable releases of those *idiomatic packages*.
+
+Checklist for completing this phase:
+
+* |chkbx| The *development test VM* is running an up-to-date development version of Granta MI.
+* |chkbx| CI for all *idiomatic packages* is passing.
+* |chkbx| *Idiomatic package* documentation has been reviewed to update changes in dependencies, for example Python
+  version support, minimum Granta MI version.
+* |chkbx| ``release`` branches have been created for *auto-generated packages* and *idiomatic packages*.
+* |chkbx| Stable *auto-generated packages* are available on PyPI.
+* |chkbx| Internal packages with runtime dependencies on *auto-generated packages* depend on the stable PyPI releases.
+* |chkbx| *Idiomatic packages* depend on stable releases of *auto-generated packages* only.
+* |chkbx| Release candidate *idiomatic packages* are available on PyPI.
+* |chkbx| Internal packages with runtime dependencies on *idiomatic packages* depend on the release candidate PyPI
+  releases.
+
+Phase 3: Pre-release
+~~~~~~~~~~~~~~~~~~~~
+
+#. Update the *production test VM* to run the Granta MI release validated during the hardening phase.
+#. Update the latest *idiomatic package* ``release`` branches created during the hardening phase to run CI against the
+   *production test VM*.
+#. Update the *PyGranta meta-package* ``main`` branch to depend on the public *idiomatic packages* release candidates
+   and *auto-generated packages* stable releases.
+#. Create a ``release`` branch for the *PyGranta meta-package* and publish as a release candidate to public PyPI.
+#. Create a Pull Request in the PyAnsys meta-package repository to move the PyGranta dependency to the release candidate
+   published in the preceding step.
+#. Manually test the *PyGranta meta-package* release candidate against the Granta MI release candidate.
+
+Testing should occur up to a week before the availability of Granta MI to customers.
+
+Checklist for completing this phase:
+
+* |chkbx| The *production test VM* is running the release version of Granta MI.
+* |chkbx| CI for all *idiomatic packages* is passing.
+* |chkbx| Release candidate *PyGranta meta-package* is available on PyPI.
+* |chkbx| The PyAnsys meta-package depends on the *PyGranta meta-package* release candidate.
+* |chkbx| Manual testing has been completed against the *PyGranta meta-package* and all *idiomatic packages*.
+
+Phase 4: Release
+~~~~~~~~~~~~~~~~
+
+Within the final week before Granta MI is available to customers, stable versions of the packages should be released:
+
+#. Publish *idiomatic packages* as stable releases to public PyPI.
```

Comment:
I think we should add something in here about the project we have, and reviewing the project before releasing to make sure there's nothing we want to implement.